### PR TITLE
Clean up old documentation and stray files throughout repo

### PR DESCRIPTION
**Pull request by Karl-G1**
### What does this Pull Request accomplish?

Cleans up documentation and extra files throughout the DSF repo:
- Delete legacy build files
- Delete empty/old docs
- Delete VI Package of LVMM
- Move manual build instructions to markdown doc
- Update readme

### Why should this Pull Request be merged?

Avoid confusing developers with outdated documentation

### What testing has been done?

None

### CODE REVIEW

File: Docs/Manual Build Instructions.md [hunk 1]
```diff
@@ -0,0 +1,30 @@
+# Manual Build Instructions for the Data Sharing Frameowrk
+These steps will allow you to manually build all of the components necessary to use the Data Sharing Framework Custom Device. This is not recommended for end-users of the Custom Device; the [Releases page](https://github.com/ni/niveristand-data-sharing-framework-custom-device/releases) should be used to find the latest builds of the Custom Device for supported versions of VeriStand.
+
+For each Target type:
+
+1.	Build DSF Core PPL
+	1. 	Open the target build project in [repo root]\Core
+	1. 	Execute the build spec
+	1. 	The Core PPL will be output to C:\Builds\Data Sharing Framework\<Target>\Core
```
Comment (rtzoeller) [inline Docs/Manual Build Instructions.md:9]: Does it still output to `C:\Builds`?

Comment (rtzoeller) [inline Docs/Manual Build Instructions.md:9]: GitHub's Markdown engine seems to be eating the slash between "Framework" and "<Target>" when rendering. I'm guessing putting the paths in backticks (e.g. `C:\Builds\Data Sharing Framework\<Target>\Core`) will fix it.

File: Docs/Manual Build Instructions.md [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+	1. 	Check that nothing is broken by mass compiling the project (right-click project > Mass compile). Things will be broken if the Core PPL type does not match the target type
+	1. 	Execute the following builds:
+		1. Windows
+			1. Configuration Release
+			1. Engine Release
+		1. Other
+			1. Engine Release
+1. (Optional) Copy Component PPLs built in step 2 to the custom device folder. This step only matters if you want to run the custom device with actual components.
+	1. 	Navigate to the built VeriStand custom device folder.
+		> C:\Users\Public\Documents\National Instruments\NI VeriStand 20XX\Custom Devices\Data Sharing Framework
```
Comment (rtzoeller) [inline Docs/Manual Build Instructions.md:28]: This is causing them to be rendered as quotes. Do you want backticks?

File: Docs/Manual Build Instructions.md (hunk 1 above)
Comment (dbendele) [reply Docs/Manual Build Instructions.md:9]: No. the built PPL outputs to ../Built/Core/<target>/DSF Core.lvlibp

Comment (Karl-G1) [reply Docs/Manual Build Instructions.md:9]: I didn't re-write the content. Mostly just formatted it. 
Let me update this with all the finalized build layout changes, then I'll put it back up for review.

File: Docs/Manual Build Instructions.md [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+	1. 	For each target type, copy the DSF Core PPL that was the output of step 1 into [repo root]\VeriStand\Data Sharing Custom Device\Source
+	1. 	Open the custom device project for the same target type
+	1. 	Check that nothing is broken by mass compiling the project (right-click project > Mass compile). Things will be broken if the Core PPL type does not match the target type
+	1. 	Execute the following builds:
+		1. Windows
+			1. Configuration Release
+			1. Engine Release
+		1. Other
+			1. Engine Release
+1. (Optional) Copy Component PPLs built in step 2 to the custom device folder. This step only matters if you want to run the custom device with actual components.
```
Comment (dbendele) [inline Docs/Manual Build Instructions.md:26]: The Post-Build Action VI invoked at the end of each build specification will copy the built component to the custom device folder.

File: Docs/Manual Build Instructions.md [hunk 4]
```diff
@@ -0,0 +1,30 @@
...
+1.	Build DSF Core PPL
+	1. 	Open the target build project in [repo root]\Core
+	1. 	Execute the build spec
+	1. 	The Core PPL will be output to C:\Builds\Data Sharing Framework\<Target>\Core
+1.	Build DSF Components
+	1. 	For each target type, copy the DSF Core PPL that was the output of step 1 into [repo root]\Components
+	1. 	Open the target build project for each component located in [repo root]\Components
+	1. 	Check that nothing is broken by mass compiling the project (right-click project > Mass compile). Things will be broken if the Core PPL type does not match the target type
+	1. 	Execute the build spec
+	1. 	The Component PPLs will be output to C:\Builds\Data Sharing Framework\<Target>\Components
```
Comment (dbendele) [inline Docs/Manual Build Instructions.md:15]: For each target, the build specification will create Component PPLs and copy them to the following locations:
   1. `<repository>\Components\Built\<target>\Components`
   1. `<Public Documents>\National Instruments\NI VeriStand <YYYY version>\Custom Devices\Data Sharing Framework\<target>\Components`

**Pull request merged**